jetty-ee9/jetty-ee9-nested/src/test/java/org/eclipse/jetty/ee9/nested/NcsaRequestLogTest.java

```diff
@@ -326,10 +351,10 @@ public void testUseragentWithout(String logType) throws Exception
         setup(logType);
         testHandlerServerStart();
 
-        _connector.getResponse("GET http://[:1]/foo HTTP/1.1\nReferer: http://other.site\n\n");
+        _connector.getResponse("GET /foo HTTP/1.1\nHost: localhost\nReferer: http://other.site\n\n");
```

Comment:
If no `Host` header the `HttpParser` will throw and no headers are reported on the request.

Reply:
So the request, now in error, never reaches the RequestLog?

Reply:
It reaches the request log in all cases. However if there was the error due to the `Host` header being absent, then the headers are completely empty.